frontend/src/edit/wallpaperingMenu.jsx

```diff
@@ -47,8 +47,10 @@ export default function WallpaperingMenu(props) {
     return newPatterns;
   };
 
-  useEffect(async () => {
-    await updatePatterns();
+  useEffect(() => {
+    (async () => {
+      await updatePatterns();
+    })();
```

Comment:
Hey Dave, I've seen you do this a few times, what's the effect of this change? Curious from a learning perspective.

Reply:
Basically it's just appeasing React18. Here's a good explanation https://stackoverflow.com/a/53332372/6850491

I'm not sure if there are any functional differences, but there would be if we ever needed a cleanup function to run after the useEffect callback.